### PR TITLE
Finalised UserGuide

### PR DESCRIPTION
**Pull request by nusliuyifan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -259,10 +163,12 @@ _Details coming soon ..._
...
+|---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| **Add Event**                               | `addev -ev <event name>` <br> e.g., `addev -ev Orientation camp`                                                      |
+| **Delete Event**                            | `delev <index>` <br> e.g., `delev 1`                                                                                  |
+| **Add Participant**                         | `addp -n <participant name> -p <phone number> -e <email>` <br> e.g., `addp -n David -p 98987676 -e david@example.com` |
+| **Selecting Event**                         | `sel <event index>` <br> e.g., `sel                                                                                   |
+| **Deselecting Event**                       | `desel` deselect the current selected event                                                                           |
+| **Deleting a participant from global list** | `del 3` delete the 3rd participant from the global participant list                                                   |
```
Comment (taeewonnn) [inline docs/UserGuide.md:173]: Change the command summary to the original format n/ p/... etc

File: docs/UserGuide.md [hunk 2]
```diff
@@ -185,63 +146,6 @@ Deselects the selected event and returns to the global participant list.
 - When no event is selected, `delp 9` deletes the 9th participant completely.
 - `delp 9` after `sel 3` removes the 9th participant from the 3rd event's participant list.
```
Comment (taeewonnn) [inline docs/UserGuide.md:147]: make evryth revert back to its original formatting

**Review by taeewonnn — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (nusliuyifan) [reply docs/UserGuide.md:147]: 😇

**Review by taeewonnn — APPROVED** [review]
Some minor changes need to be done but ill do it. rest lgtm

**Pull request merged**